lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -0,0 +1,97 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_runtime_pair")
+load("@subpar//:subpar.bzl", "par_binary")
+load("@python_deps//:requirements.bzl", "requirement")
+
+py_runtime(
```

Comment:
Questions, rather than review comment :)
1. does the following tool chain config below take python3 from the underlying system? 
2. Am I correct in understanding that we'll move this out to be shared by other services if we add others?

Reply:
1. The toolchain uses the python interpreter specified in WORKSPACE.bazel (line 150 and following), thus it is independent from the python installation(s) from the underlying system.

2. I think that makes sense. I can move into a BUILD file on a higher level. Otherwise we would have duplicate lines of code.

Reply:
so we could technically uninstall python in the underlying system and this would still work?

Reply:
re: 2, I don't have a preference for doing this now vs later. I just wanted to make sure I was understanding this correctly.

Reply:
Yes, it should work without a pre-installed python.

Reply:
Can we then remove python from the bazelbase container then? I'd like to keep that container as clean as possible.

Reply:
I'm currently trying to build and run mobilityd inside the bazelbase container without an installed python 3.

Reply:
I could build mobilityd inside the bazelbase container without a pre-installed python version. Two additional  entries are required in the dockerfile. The system library `libffi-dev` and a symlink that refers to the installed python interpreter.  The former is needed for the installation of the python toolchain with all dependencies and the latter for the build  process of the  gprc python package .